san/backtest.py

```diff
@@ -2,53 +2,44 @@
 import pandas as pd
 import numpy as np
 import san
+import matplotlib.pyplot as plt
```

Comment:
Should we put `matplotlib` in https://github.com/santiment/sanpy/blob/master/setup.py#L20 ?

Reply:
Yep, sure
Thanks!